estate_account/__manifest__.py

```diff
@@ -0,0 +1,13 @@
+{
+    'name' : 'Real Estate Account',
+    'version' : '1.0',
+    'author' : 'odoo',
+    'depends' : ['estate', 'account'],
+    'data' : [
+        'views/estate_property_views.xml',
+        'views/account_invoice_views.xml',
+    ],
+    'license': 'LGPL-3',
+    'installable': True,
+    'application': True
```

Comment:
```suggestion
    'auto_install': True,
```
Here you're creating a bridge module that contains functionality that we'd like to be present whenever both `account` and `estate` are installed. 
This makes it install itself upon this condition instead the user having to look for it in the interface.